.github/workflows/sub_lint.yml

```diff
@@ -77,7 +77,73 @@ on:
         type: boolean
         required: false
       cpplintFilters:
-        default: -whitespace-*
+        default: >-
+          '
+          +build/deprecated,
+          +build/endif_comment,
+          +build/explicit_make_pair,
+          +build/forward_decl,
```

Comment:
If this means to disallow forward declarations then this rule sucks. Forcing to always include the corresponding header file makes the already long build process to take a lot of extra time.

Reply:
This isn't a change from our current state: all of the checks with a "+" in front of them were already enabled, I'm just listing them out so that we can see what's available, and make better decisions about what to switch off.